subworkflows/local/contrasts_check.nf

```diff
@@ -0,0 +1,23 @@
+//
+// Check input contrastsheet and get read channels
+//
+
+include { CONTRASTSHEET_CHECK } from '../../modules/local/contrastsheet_check'
```

Comment:
Can this not just be a module?

Reply:
We developed it as a sub workflow because we feel there will be some additional logic we need to handle when we implement additional parts of the contrast sheet check. Similar to how the sample sheet check module is within the input check sub workflow.